content/articles/guide-to-run-react-native-app-with-typescript/index.md

```diff
@@ -0,0 +1,244 @@
+    }
+]
+```
+
+From above, we are exporting an array of posts. Each post has the following data:
+
+- `id`: A unique integer value.
+- `title`: The posts title.
+- `excerpt`: Small description of the title.
+- `content`: Dummy article content.
+
+### Fetching posts
+
+To implement the functionality of fetching the posts, we will work on `components/Posts.tsx`.
+
+- Import  `StyleSheet` and the dummy posts:
+
+```ts
+import {StyleSheet} from 'react-native';
+import posts from '../lib/posts';
+```
+
+- Define a post's structure:
+
+```ts
+interface Post {
+    id: number;
+    title: string;
+    excerpt: string;
+    content: string;
+}
+```
+
+- Map the posts to the view:
+
+```ts
+export default function Posts(){
+    return(
+        <View>
+            {
+                posts.map((post:Post)=>{
+                    return (
+                        <View key={post.id} style={styles.postCard}> 
+                            <Text style={styles.postTitle}>{post.title}</Text>
+                            <Text style={styles.postExcerpt}>{post.excerpt}</Text>
+                        </View>
+                    )
+            })}
+        </View>
+    )
+}
+```
+
+- Append the styles:
+
+```ts
+const styles = StyleSheet.create({
+    postCard:{
+        width:'100%',
+        padding:10,
+    },
+    postTitle:{
+        fontSize:18,
+        fontWeight:'bold',
+    },
+    postExcerpt:{
+        fontSize:15,
+        color:'#666',
+    }
+});
+```
+
+In `App.tsx`, edit the styles as follows, removing the centre alignment:
+
+```ts
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    paddingTop: 40,
+  },
+});
+```
+
+Your posts should now be shown on your screen:
+
+![posts-screen](/engineering-education/guide-to-run-react-native-app-with-typescript/posts-screen.jpg)
+
+### Conclusion
+This guide walked you through a reflective introduction to Typescript and React Native. For better understanding, we have created a whole application that uses React Native to run the Typescript code.
```

Comment:
Include any next or final steps summarizing the tasks the audience just completed with possible next steps.
Also, include a link to the Github repo or live site, if you can.